packages/shared/src/components/utilities.tsx

```diff
@@ -3,6 +3,8 @@ import classed from '../lib/classed';
 import styles from './utilities.module.css';
 import ArrowIcon from '../../icons/arrow.svg';
 
+export type TargetTypes = '_blank' | '_parent' | '_self' | '_top';
```

Comment:
I think this type already exists out of the box, isn't it?

Reply:
Oh yep! 👏